### PR TITLE
add contributing file

### PR DESCRIPTION
**Pull request by hindujachennupati**
#### Description

Github issue: (https://github.com/sourcefuse/terraform-aws-arc-document-db/issues/9)

#### Objective

Please adhere to this format when creating or customizing ARC modules

https://github.com/sourcefuse/terraform-aws-arc-github-iam/blob/main/docs/module-usage-guide/README.md

Please include a [CONTRIBUTING.md](https://github.com/sourcefuse/terraform-aws-arc-github-iam/blob/main/CONTRIBUTING.md)

#### Type

- [ ] Bug Fix
- [ ] New Feature
- [ ] Performance Tweaks
- [ ] Style Tweaks
- [ ] Code Refactoring
- [x] Documentation
- [ ] Tests

#### Work Done

- Add usage guide and CONTRIBUTING.md
- update README.md

### CODE REVIEW

File: .github/workflows/docs.yaml [hunk 1]
```diff
@@ -22,3 +22,29 @@ jobs:
           user_email: "github-actions@github.com"
           user_name: ${{ github.actor }}
           commit_message: ${{ github.event.head_commit.message }}
+      - name: Pushes static folder
+        uses: cpina/github-action-push-to-another-repository@main
+        env:
```
Comment (mayank0202) [inline .github/workflows/docs.yaml:27]: where is the static folder? if its not there then remove it

File: example/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Contributing to AWS ARC Network
+Thank you for considering contributing to AWS ARC Network! We appreciate your time and effort.
+To ensure a smooth collaboration, please take a moment to review the following guidelines.
+
+## How to Contribute
+1. Fork the repository to your own GitHub account.
+2. Clone the repository to your local machine.
+   ```bash
+   git clone https://github.com/<your_organization>/<your_terraform_module>.git
```
Comment (mayank0202) [inline example/CONTRIBUTING.md:9]: why you have added the same file in example folder? remove it

File: .github/workflows/docs.yaml (hunk 1 above)
Comment (hindujachennupati) [reply .github/workflows/docs.yaml]: Removed

File: example/CONTRIBUTING.md (hunk 2 above)
Comment (hindujachennupati) [reply example/CONTRIBUTING.md]: done

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+## How to Contribute
+1. Fork the repository to your own GitHub account.
+2. Clone the repository to your local machine.
+   ```bash
+   git clone https://github.com/<your_organization>/<your_terraform_module>.git
+   ```
+3. Create a new branch for your feature / bugfix.
+   ```bash
+   git checkout -b feature/branch_name
+   ```
```
Comment (vijay-stephen) [inline CONTRIBUTING.md:14]: Add pre-commit install

**Review by vijay-stephen — CHANGES_REQUESTED** [review]

File: docs/module-usage-guide/README.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+### Input Variables
+
+For a list of input variables, see the README [Inputs](https://github.com/sourcefuse/terraform-aws-arc-document-db?tab=readme-ov-file#inputs) section.
+
+### Output Values
+
+For a list of outputs, see the README [Outputs](https://github.com/sourcefuse/terraform-aws-arc-document-db?tab=readme-ov-file#outputs) section.
+
+## Module Usage
```
Comment (mayank0202) [inline docs/module-usage-guide/README.md:64]: Please add the usage

File: CONTRIBUTING.md (hunk 3 above)
Comment (hindujachennupati) [reply CONTRIBUTING.md:14]: done

File: docs/module-usage-guide/README.md (hunk 4 above)
Comment (hindujachennupati) [reply docs/module-usage-guide/README.md:64]: done

**Review by mayank0202 — APPROVED** [review]

**Pull request merged**